packages/mobile/src/screens/profile-screen/ArtistRecommendations/ArtistRecommendations.tsx

```diff
@@ -102,12 +94,15 @@ export const ArtistRecommendations = (props: ArtistRecommendationsProps) => {
   const artistsToFollow = useSelector<CommonState, User[]>((state) =>
     Object.values(getUsers(state, { ids: idsToFollow || [] }))
   )
-  const suggestedArtists = useSelector((state) =>
-    selectSuggestedFollowsUsers(state, { id: user_id })
-  )
+
+  const { data: suggestedArtists } = useRelatedArtists({
+    artistId: user_id,
+    filterFollowed: true
+  })
+
   useEffect(() => {
-    if (!isEmpty(suggestedArtists)) {
-      setIdsToFollow(suggestedArtists.map((user) => user.user_id).slice(0, 5))
+    if (suggestedArtists && !isEmpty(suggestedArtists)) {
```

Comment:
does isEmpty not check truthiness? :(

Reply:
yeah it's so typescript is happy when i unconditionally .map suggestedArtists